.storybook/polyfill-globals.js

```diff
@@ -0,0 +1,3 @@
+import { resetGlobals } from './utils/resetGlobals';
```

Comment:
Please add a file header ☝️

Reply:
Added @aaemnnosttv

Reply:
Updated @aaemnnosttv  I also updated the year in `main.js`